rackcity/api/serializers/it_model_serializers.py

```diff
@@ -46,13 +40,7 @@ class BulkITModelSerializer(serializers.ModelSerializer):
     Serializes all fields on ITModel model according to the format required
     for bulk export.
     """
-    ethernet_ports = RCIntegerField(
-        source='num_ethernet_ports',
-        allow_null=True,
-        max_value=2147483647,
-        min_value=0,
-        required=False
-    )
```

Comment:
I would recommend keeping this serializer unchanged since we're not sure how this is going to affect the implementation of model bulk imports yet. I'd say it's likely that we're going to get rid of this field on the serializer, but I think it'd be better to keep this for now and only take it out once we're sure